doc/source/whatsnew/v0.18.2.txt

```diff
@@ -16,20 +16,38 @@ Highlights include:
 
 .. _whatsnew_0182.new_features:
 
-New features
-~~~~~~~~~~~~
-
+Series.tolist() updated to use Python types
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
make this ``^^^`` be the same length as the title.

``.tolist()`` will now return Python types.